.github/workflows/pull-request.yml

```diff
@@ -33,12 +36,32 @@ jobs:
       - uses: ./.github/actions/install-cache
       - uses: ./.github/actions/build-cache
       - uses: ./.github/actions/turbo-cache
+      - name: Configure AWS credentials (dev)
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.NIHR_GITHUB_ACTIONS_IAM_ID }}
+          aws-secret-access-key: ${{ secrets.NIHR_GITHUB_ACTIONS_IAM_SECRET }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Get secrets from Secrets Manager
+        uses: aws-actions/aws-secretsmanager-get-secrets@v2
+        with:
+          secret-ids: |
+            crnccd-secret-dev-se-app-config
+          parse-json-secrets: true
       - name: Build
         # Inject the minimum amount of env vars needed to build the project
         run: |
           touch apps/web/.env.local
           echo DATABASE_URL=${{ secrets.DATABASE_URL }} >> apps/web/.env.local
+          echo CONTENTFUL_BANNER_ENTRY_ID=${{ env.CRNCCD_SECRET_DEV_SE_APP_CONFIG_CONTENTFUL_BANNER_ENTRY_ID }} >> apps/web/.env.local
+          echo CONTENTFUL_SPACE_ID=${{ env.CRNCCD_SECRET_DEV_SE_APP_CONFIG_CONTENTFUL_SPACE_ID }} >> apps/web/.env.local
+          echo CONTENTFUL_ACCESS_TOKEN=${{ env.CRNCCD_SECRET_DEV_SE_APP_CONFIG_CONTENTFUL_ACCESS_TOKEN }} >> apps/web/.env.local
+          echo CONTENTFUL_ENVIRONMENT=${{ env.CRNCCD_SECRET_DEV_SE_APP_CONFIG_CONTENTFUL_ENVIRONMENT }} >> apps/web/.env.local
+          echo CONTENTFUL_PREVIEW_MODE=${{ env.CRNCCD_SECRET_DEV_SE_APP_CONFIG_CONTENTFUL_PREVIEW_MODE }} >> apps/web/.env.local
+          echo CONTENTFUL_PREVIEW_ACCESS_TOKEN=${{ env.CRNCCD_SECRET_DEV_SE_APP_CONFIG_CONTENTFUL_PREVIEW_ACCESS_TOKEN }} >> apps/web/.env.local
           echo NEXT_PUBLIC_APP_ENV=${{ vars.APP_ENV }} >> apps/web/.env.local
+          echo CONTENTFUL_BANNER_ENTRY_ID=test>> apps/web/.env.local
```

Comment:
Should this be appending 'test' to this file?

Reply:
yeah these probably can be loaded into a .env.test file but I was just following the convention there already rather than figuring out if it would work.  I will test it with .env.test

Reply:
I guess it just needs to be the actual entry ID: `echo CONTENTFUL_BANNER_ENTRY_ID=<real entry ID> >> apps/web/.env.local`